tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "usage: $0 <repository-root> <tool-name> [...]"
```

Comment:
<override-scripts-folder-path>.

---
In reply to: [110402206](https://github.com/ravimeda/corefx/pull/9#discussion_r110402206) [](ancestors = 110402206)